style.css

```diff
@@ -366,3 +405,149 @@ textarea {
   justify-content: center;
   border-radius: 5px;
 }
+
+@media all and (max-width: 767px) {
+  .nav-link {
+    display: none;
+  }
+
+  .header {
+    max-width: 300px;
+  }
+}
+
+@media all and (min-width: 768px) {
+  header {
+    color: white;
+    background: no-repeat url(images/headerbg.svg);
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .header,
+  .header-content,
+  .header-footer,
+  .social {
+    padding-left: 260px;
+  }
+
+  .header {
+    max-width: 630px;
+  }
+
+  .header-content {
+    max-width: 920px;
+  }
+
+  .my-logo {
+    padding-left: 145px;
+  }
+
+  .container-main {
+    margin: 0;
+    padding-bottom: 10rem;
+  }
+
+  .social-icons {
+    padding-right: 25rem;
+  }
+
+  .card {
+    width: 65%;
+  }
+
+  .card-content-1 h3 {
+    margin-bottom: 0;
+  }
+
+  .card-container-1 {
+    display: flex;
+    flex-direction: row-reverse;
+    column-gap: 1.5rem;
+  }
+
+  .card-content-1 {
+    display: flex;
+    flex-direction: column;
+    justify-self: start;
+    margin-left: 0;
+    padding-left: 2rem;
+    align-items: flex-start;
+    margin-right: auto;
+    padding-top: 2rem;
+  }
+
+  .card-container-1 img {
+    padding-right: 2rem;
+    padding-top: 3rem;
+  }
+
+  .card-container {
+    display: flex;
+    flex-direction: row;
+    column-gap: 4rem;
+    padding-top: 2rem;
+  }
+
+  .card-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: left;
+    align-items: left;
+    column-gap: 2rem;
+    padding-left: 1rem;
+  }
+
+  .about-container {
+    display: flex;
+    flex-direction: row;
+    column-gap: 3rem;
+    justify-content: space-between;
+  }
+
+  .about-content {
+    display: flex;
+    flex-direction: column;
+    padding-left: 100px;
+    max-width: 600px;
+  }
+
+  .about-content p {
+    font-size: 18px;
+    font-weight: 500;
+  }
+
+  .btn-icon-language {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    margin-right: 6rem;
+  }
+
+  .stack {
+    display: flex;
+    flex-direction: row;
+    gap: 1.2rem;
+  }
+
+  .stack li {
+    display: flex;
+    width: 140px;
+    height: 120px;
+    padding-right: 3rem;
+  }
+
+  .display {
+    display: block;
+  }
+
+  .contact-container p {
+    max-width: 562px;
+  }
```

Comment:
- [ ] I recommend removing the border from the contact form inputs so that your design matches the Figma design.

- Your design:
<img width="248" alt="image" src="https://user-images.githubusercontent.com/77190203/171412154-4689cd47-c792-4f50-ab06-ea28c1f19760.png">

- Figma design:
<img width="220" alt="image" src="https://user-images.githubusercontent.com/77190203/171412269-f0da073e-6375-406e-a27b-41adc00004d1.png">